frontend/tests/features/wheresMyGene/landingPage.ts

```diff
@@ -0,0 +1,109 @@
+import { expect, Page, test } from "@playwright/test";
+import { TEST_URL } from "../../common/constants";
+import { ROUTES } from "src/common/constants/routes";
+
+const { describe } = test;
+const ALERT =
+  "We would appreciate your feedback, please fill out a quick survey";
+
+const SURVEY_LINK = "https://airtable.com/shrLwepDSEX1HI6bo";
+const ADD_TISSUE = "add-tissue";
+const ADD_GENES = "add-genes";
+const EXPLORE_GENE_EXPRESSION = "explore-gene-expression";
+const LEGEND_WRAPPER = "legend-wrapper";
+const DOT_SIZES: Record<number, number> = {
+  0: 4,
+  1: 9,
+  2: 12,
+  3: 14,
+  4: 16,
+};
+async function checkDotSize(index: number, page: Page) {
+  expect(
+    page
+      .locator('[id="expressed-in-cells-dots"]')
+      .locator("span")
+      .nth(0)
+      .getAttribute("size")
+  ).toBe(DOT_SIZES[index]);
+}
+describe("Tests for Gene Expression page", () => {
+  test.beforeEach(async ({ page }) => {
+    await Promise.all([
+      page.waitForResponse(
+        (resp: { url: () => string | string[]; status: () => number }) =>
+          resp.url().includes("/wmg/v1/filters") && resp.status() === 200
+      ),
+      page.goto(`${TEST_URL}${ROUTES.WHERE_IS_MY_GENE}`),
+    ]);
+  });
+
+  test("Should verify main panel components", async ({ page }) => {
+    // +Tissue button
+    await expect(page.getByTestId(ADD_TISSUE).first()).toBeVisible();
```

Comment:
seems like the use of `.first()` is discouraged in https://playwright.dev/docs/locators#strictness. Can we narrow the search scope, maybe by scoping it to the `<GeneSearchBar />` component? 

Or maybe this issue will naturally go away by avoiding conflict `data-testid` name in `frontend/src/views/WheresMyGene/components/GetStarted/index.tsx`!

Thank you!